### ISSUE TITLE
Tags are not accessible using the keyboard: A11y_.NET 9 template_.NET 9 template_Project_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26860]
### GitHub Tags
#A11yTCS; #A11ySev1; #Win32; #WCAG2.1.1; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Keyboard; #MAC;#AILimited;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:
1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till project section, and press enter on any project, Project screen will open,
4. Tab till tags.
5. Observe that tags are not accessible using the keyboard keys.
 
### Actual Result:
Observe that tags are not accessible using the keyboard keys.

### Expected Result:
Tags should be accessible using the keyboard keys.

### User Impact:
keyboard users will not be able to access the controls and will not be able to access the other functionality.

### Attachment:

https://github.com/user-attachments/assets/c33e95f6-6552-49bb-8e46-e323f54f845a

**Comment by AnKushSingh05** [comment]
Issue is fixed
